apim/3.x/templates/api/api-configmap.yaml

```
@@ -88,6 +88,12 @@ data:
         url: "https://{{index .Values.api.ingress.management.hosts 0 }}{{ .Values.api.ingress.management.path }}"
       ui:
         url: "https://{{index .Values.ui.ingress.hosts 0 }}{{ regexFind "/[a-zA-Z]+" .Values.ui.ingress.path }}"
+      {{- if (((.Values.ui).analytics).pendo).enabled }}
+      analytics:
+        pendo:
+          enabled: {{ .Values.ui.analytics.pendo.enabled }}
+          apiKey: {{ .Values.ui.analytics.pendo.apiKey }}
```

Do you think it's better to replace the content in analytics by something like this ?

```suggestion
 {{ toYaml .Values.ui.analytics | indent 8 }}
```

---

I don't really have an opinion, it makes the possible configurations less explicit but more flexible. 
As you wish ? ;)

---

As there is no real rule and not some much lines we can let it like it is. :)